### PR TITLE
chore: add construct documentation to the README

### PR DESCRIPTION
**Pull request by sharkinsspatial**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
-# pgSTAC CDK construct
+# eoAPI CDK Construct
+
+eoapi-cdk is a package of [AWS CDK constructs](https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/constructs-best-practices.html) designed to encapsulate eoAPI runtime code and best practices as simple reusable components.
```
Comment (vincentsarago) [inline README.md:3]:
```suggestion
eoapi-cdk is a package of [AWS CDK constructs](https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/constructs-best-practices.html) designed to encapsulate eoAPI services and best practices as simple reusable components.

```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+
+Authentication for the STAC Ingestor API can be configured with JWTs authenticated by JWKS.  To learn more about securing FastAPI applications with this approach see [Securing FastAPI with JWKS (AWS Cognito, Auth0)](https://alukach.com/posts/fastapi-rs256-jwt/).
+
+A sample Cognito-based authentication system is available at [aws-asdi-auth](https://github.com/developmentseed/aws-asdi-auth).
+
+### [STAC API](https://developmentseed.org/cdk-pgstac/#pgstacapilambda-)
+A STAC API implementation using [stac-fastapi](https://github.com/stac-utils/stac-fastapi) with a [pgSTAC backend](https://github.com/stac-utils/stac-fastapi-pgstac). Packaged as a complete runtime for deployment with API Gateway and Lambda.
+
+### [pgSTAC Titiler API](https://developmentseed.org/cdk-pgstac/#titilerpgstacapilambda-)
+A complete dynamic tiling API using [https://github.com/stac-utils/titiler-pgstac] to create dynamic mosaics of assets based on [STAC Search queries](https://github.com/radiantearth/stac-api-spec/tree/master/item-search).  Packaged as a complete runtime for deployment with API Gateway and Lambda and fully integrated with the pgSTAC Database construct.
```
Comment (vincentsarago) [inline README.md:35]: This is great 🙏 

I would just change the order for the services 

- pgstac
- stac api
- titiler-pgstac 
- stac-ingestor
- bastion

Comment (vincentsarago) [inline README.md:35]:
```suggestion
A complete dynamic tiling API using [stac-utils/titiler-pgstac](https://github.com/stac-utils/titiler-pgstac) to create dynamic mosaics of assets based on [STAC Search queries](https://github.com/radiantearth/stac-api-spec/tree/master/item-search).  Packaged as a complete runtime for deployment with API Gateway and Lambda and fully integrated with the pgSTAC Database construct.

```

**Review by vincentsarago — APPROVED** [review]
✨

File: README.md (hunk 1 above)
Comment (emileten) [reply README.md:3]: If I am understanding your comment correctly vincent, this relates to https://github.com/developmentseed/cdk-pgstac/issues/4 (we currently have the stac ingestor runtime stored here, whereas it ideally should be maintained separately)

**Pull request merged**